lerna.json

```diff
@@ -12,5 +12,13 @@
     "packages/groq",
     "packages/sanity"
   ],
-  "version": "3.90.0"
+  "version": "3.90.0",
+  "command": {
+    "version": {
+      "changelogPreset": "conventionalcommits",
+      "message": "chore(release): publish %s",
+      "changelogIncludeCommitsClientLogin": " by %a (%e)",
```

Comment:
@bjoerge  If you're only using Git details, then it would be better to use [`--changelog-include-commits-git-author`](https://github.com/lerna-lite/lerna-lite/tree/main/packages/version#--changelog-include-commits-git-author-msg). The difference is that `--changelog-include-commits-client-login` does a GraphQL API fetching (which GitHub set a limit of 200 commits/fetch if I remember correctly) in order to get GitHub username login (`%l`) but with the `--changelog-include-commits-git-author` it's pulled directly from Git (no fetching required, and so a much lighter approach since `%a` and `%e` are both available in Git commits).

Side note I'm the Lerna-Lite maintainer and I added these 2 flags.

Reply:
Thanks @ghiscoding! And thank you for the excellent work on lerna-lite, love the focus of the project. Was glad to see how smooth it was to set it up for us. Much appreciated! ❤️ 

Opened a PR that changes the config to instead use `--changelog-include-commits-git-author` over [here](https://github.com/sanity-io/sanity/pull/9558).